app/assets/javascripts/controllers/container_dashboard/card/card-module.js

```diff
@@ -0,0 +1 @@
+angular.module('miq.card', []);
```

Comment:
> is this one liner file necessary?

@abonas Its nicer to have a file that describes a module than to have to search for where it is in other cards.